src/System.Security.Cryptography.Algorithms/tests/HmacSha384Tests.cs

```diff
@@ -20,6 +20,19 @@ protected override HashAlgorithm CreateHashAlgorithm()
 
         protected override int BlockSize { get { return 128; } }
 
+#if netcoreapp11
```

Comment:
Actually, I think the conditionals should be `netstandard17`... `netcoreapp11` is for API in .NET Core that isn't in .NET Standard.  (Applies to the other PRs as well)